plasmapy/formulary/parameters.py

```diff
@@ -533,16 +540,141 @@ def ion_sound_speed(
+
+    ndim : `int`
+        Dimensionality (1D, 2D, 3D) of space in which to calculate thermal
+        speed. Valid values are ``1``, ``2``, or ``3``.
+
+    Raises
+    ------
+    `ValueError`
+        If ``method`` or ``ndim`` are not a valid value.
+
+    Notes
+    -----
+
+    For a detailed explanation of the different coefficients used to calculate
+    the therml speed, then look to the :ref:`Notes <thermal-speed-notes>` section
+    for  `~plasmapy.formulary.parameters.thermal_speed`.  The possible return
+    values are listed the table
+
+    .. table:: Thermal speed :math:`v_{th}` coefficients.
+       :widths: 2 1 1 1 1
+       :width: 100%
+
+       +--------------+------------+---------------+---------------+---------------+
+       | ↓ **method** | **ndim** → | ``1``         | ``2``         | ``3``         |
+       +--------------+------------+---------------+---------------+---------------+
+       | ``"most_probable"``       | .. math::     | .. math::     | .. math::     |
+       |                           |    0          |    1          |    \sqrt{2}   |
+       +--------------+------------+---------------+---------------+---------------+
+       | ``"rms"``                 | .. math::     | .. math::     | .. math::     |
+       |                           |    1          |    \sqrt{2}   |    \sqrt{3}   |
+       +--------------+------------+---------------+---------------+---------------+
+       | ``"mean_magnitude"``      | .. math::     | .. math::     | .. math::     |
+       |                           |    \sqrt{2/π} |    \sqrt{π/2} |    \sqrt{8/π} |
+       +--------------+------------+---------------+---------------+---------------+
+       | ``"nrl"``                 | .. math::                                     |
+       |                           |    1                                          |
+       +--------------+------------+---------------+---------------+---------------+
+
+    Examples
+    --------
+    >>> thermal_speed_coefficients(method="most_probable", ndim=3)
+    1.414213...
+    """
+    _coefficients = {
+        (1, "most_probable"): 0,
+        (2, "most_probable"): 1,
+        (3, "most_probable"): np.sqrt(2),
+        (1, "rms"): 1,
+        (2, "rms"): np.sqrt(2),
+        (3, "rms"): np.sqrt(3),
+        (1, "mean_magnitude"): np.sqrt(2 / np.pi),
+        (2, "mean_magnitude"): np.sqrt(np.pi / 2),
+        (3, "mean_magnitude"): np.sqrt(8 / np.pi),
+        (1, "nrl"): 1,
+        (2, "nrl"): 1,
+        (3, "nrl"): 1,
+    }
+
+    try:
+        coeff = _coefficients[(ndim, method)]
+    except KeyError:
+        raise ValueError(
+            f"Value for (ndim, method) pair not valid, got '({ndim}, {method})'."
+        )
+
+    return coeff
+
+
+@preserve_signature
+@njit
+def thermal_speed_lite(
+    T: numbers.Real, mass: numbers.Real, coeff: numbers.Real
+) -> numbers.Real:
+    r"""
+    The "Lite-Function" version of `~plasmapy.formulary.parameters.thermal_speed`.
```

Comment:
...  I did it this way for two reasons: (1) I'm trying to establish the terminology "lite-function" and what it means and (2) I want to push user to `thermal_speed` for the details.  In the end I think this approach would be less maintenance for us.

Maybe as a compromise I can add "lite-function" to the glossary and use that here.

Reply:
Sure!  Good points.  I do quite like the idea of adding "lite-function" to the glossary so we can link to it with the `:term:` role.  In any case, I'm happy to go with whatever you decide on this.

Reply:
lite-function is now added to the glossary.  I also updated the description under the `automodapi` "Lite-Functions" section to point users to the lite-function glossary entry for details.  Hopefully, this will minimize maintenance down the line if we modify the scope of lite-functions.